kinder/ci/tools/update-workflows/templates/workflows/dryrun-tasks.yaml

```
@@ -49,16 +49,29 @@ tasks:
     - --worker-nodes={{ .vars.workerNodes }}
     - --loglevel=debug
   timeout: 5m
-- name: init-dryrun
+- name: dryrun-without-cluster
   description: |
-    Initializes the Kubernetes cluster in dryrun mode
+    Run kubeadm commands in dryrun mode without a real cluster
   cmd: /bin/bash
   args:
     - -c
     - |
-      docker exec {{ .vars.clusterName }}-control-plane-1 bash -c "until crictl ps &> /dev/null; do echo 'Waiting for the container runtime to be running ...'; sleep 1; done"
-      docker exec {{ .vars.clusterName }}-control-plane-1 kubeadm init --ignore-preflight-errors={{ .vars.kubeadmIgnorePreflightErrors }} --kubernetes-version={{ .vars.kubernetesVersion }} --dry-run=true --v={{ .vars.kubeadmVerbosity }} || exit 1
-      docker exec {{ .vars.clusterName }}-control-plane-1 kubeadm init --ignore-preflight-errors={{ .vars.kubeadmIgnorePreflightErrors }} --kubernetes-version={{ .vars.kubernetesVersion }} --upload-certs=true --dry-run=true --v={{ .vars.kubeadmVerbosity }} || exit 1
+      set -x
+      required_version="1.32" # dryrun without cluster is available since v1.32
+      current_version=$(echo "{{ .vars.kubernetesVersion }}" | grep -oP 'v\K\d+\.\d+')
+      if awk "BEGIN {exit !($current_version < $required_version)}"; then
```

i think this would work too
```suggestion
      if [ "$current_version" \< "$required_version" ]; then
```

tested in a local bash.

```sh
#!/bin/bash

required_version="1.32"
current_version=$(echo "v1.34.0" | grep -oP 'v\K\d+\.\d+')

if [ "$current_version" \< "$required_version" ]; then
    echo "less"
else
    echo "supported"
fi
```